test/remote/gateways/remote_ixopay_test.rb

```diff
@@ -44,53 +44,57 @@ def test_failed_purchase
   end
 
   def test_failed_authentication
-    gateway = IxopayGateway.new(username: 'baduser', password: 'badpass', secret: 'badsecret')
+    gateway = IxopayGateway.new(
+      username: 'baduser',
+      password: 'badpass',
+      secret:   'badsecret',
+      api_key:  'badapikey'
+    )
+
     response = gateway.purchase(@amount, @credit_card, {})
 
     assert_failure response
 
-    assert_equal 'Invalid Signature: Invalid authorization header', response.message
```

Comment:
Just updated the error message here now that the API key is passed in for this test as well.